.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "resources/examples"]
 	path = resources/examples
 	url = https://github.com/AntaresSimulatorTeam/Antares_Simulator_Examples.git
+[submodule "src/antares-deps"]
+	path = src/antares-deps
+	url = https://github.com/AntaresSimulatorTeam/antares-deps.git
+	branch = feature/init
```

Comment:
Will be changed to a tag when antares-deps review is done and a tag is available

Reply:
yes this will be updated when tag v1.0.0 is available